cli/pcluster/config/validators.py

```diff
@@ -48,6 +48,11 @@
 
 FSX_SUPPORTED_OSES = ["centos7", "ubuntu1604", "ubuntu1804", "alinux", "alinux2"]
 
+BASE_ADDITIONAL_IAM_POLICIES = [
+    "arn:aws:iam::aws:policy/CloudWatchAgentServerPolicy",
```

Comment:
please check if this arns are compatible with govcloud and cn regions

Reply:
You made me also notice that the ``get_partition()`` method was not detecting cn-partition. Please check the updated commit.